platform/podjs/src/manifest-parser/src/manifest.ts

```diff
@@ -5,7 +5,6 @@ import { parse as parseSemVer, SemVer, Range } from "semver";
 
 export interface MainManifest {
     readonly name: string;
-    readonly version: SemVer;
 }
 
 export interface FeatureManifest {
```

Comment:
The thing with TODOs is that someone, some time, has to actually `DO` them. `links`  and `translations` are probably no longer unknown, as far as that gets you.